ormar/queryset/actions/order_action.py

```diff
@@ -30,10 +35,16 @@ def __init__(
         if self.source_model == self.target_model and "__" not in self.related_str:
             self.is_source_model_order = True
 
+        self.nulls = nulls_ordering if nulls_ordering is not None else None
```

Comment:
that line should be simply `self.nulls = nulls_ordering` cause `is not None else None` part means that if `null_ordering` is None it's gonna be None anyway. You should also probably use `self.nulls_ordering` variable name to store the value.

Reply:
Hi @collerek 
I apologize for the delay;
I made the changes you said.